gap/attr.gi

```diff
@@ -1654,6 +1655,11 @@ function(D)
     return C;
 end);
 
+InstallMethod(DigraphAllChordlessCycles, "for a digraph",
+[IsDigraph],
+function(D) return DigraphAllChordlessCyclesOfMaximalLength(D, INTOBJ_MAX);
```

Comment:
You could make this function a lambda, and replace `INTOBJ_MAX` with `infinity`.

Reply:
How can I write a lambda function in GAP?

Reply:
I'm not sure how to use infinity here, because then I need a filter other than IsInt.

Reply:
Sounds good